### ISSUE TITLE
[a11y] [Single record] when editing, hate crimes options don't have a screen reader label

### ISSUE THREAD

**Issue by Jacklynn** [18F/crt-portal#527]
When using a screen reader, the hate crimes & human trafficking options are not labeled:
![Screen Shot 2020-05-18 at 5.36.06 PM.png](https://images.zenhubusercontent.com/5d1a3b0bcc794602e102a513/b59ee5a0-6e10-44ca-9527-2e97a2eb6d34)

They read "yes, unchecked" but I'm not able to tell what they are.

Expected the labels to be "Hate crime" and "human trafficking" respectively. So they'd read something like  "Yes, unchecked. Hate crime group"

Example:
- https://webaim.org/techniques/forms/controls#checkbox

**Comment by LindsayYoung** [comment]
FYI: These will rarely exist once we do the hate crime redo.

**Comment by cantsin** [comment]
I think the `role` attr was mis-assigned on the checkbox label -- currently it has `role="label"` which doesn't seem right to me (no such thing?). Easier to just remove that role attr altogether and then the checkbox just should work (no extra aria label needed, from what i understand). So yea, depends on if `usa_checkbox_option.html` template exists after the "hate crime and human trafficking" redo...

**Comment by LindsayYoung** [comment]
Thanks for the insight on that @cantsin! It seems like that would be taken care of in:
        https://github.com/usdoj-crt/crt-portal/pull/402

@Jkrzy is that PR good to go? It's been labeled WIP so I wasn't sure

**Comment by cantsin** [comment]
Aha. Yes, that is exactly what I was talking about. Thanks. That should fix the problem here.

**Comment by cantsin** [comment]
I think this issue is done; can we validate functionality works as intended?

**Comment by cantsin** [comment]
I was wrong; seems like this problem still persists. (Thank you @Jacklynn for checking this)

**Comment by Jkrzy** [comment]
With the revisions to these questions, https://github.com/usdoj-crt/crt-portal/pull/487, I don't think this one is applicable anymore. The form input on the edit page is a select now for `hatecrime`

**Comment by LindsayYoung** [comment]
Closing since we don't have these fields anymore
